components/header/index.styl

```diff
@@ -1,15 +1,44 @@
 .header
-  background red
   height 350px
+  position relative
+  background-image url('./images/Banner_3.jpg')
+  background-position center
+  background-size 526px
 
 .header nav
   width 100%
   height 62px
-  position absolute
-  top 0
-  left 0
+  // top 0
+  // left 0
   border-bottom 1px solid white
 
 .header svg
+  padding 10px
   polygon, path
     fill white
+
+.header-language-toggle
+  float right
+  margin 0 15px
+  position relative
+  top 50%
+  transform translateY(-50%)
```

Comment:
Works for me! Other ways to accomplish nudging things around are `margin-top: -10px` or `top calc(50% - 10px)`. So many tricks.